ext/web/internal.d.ts

```diff
@@ -43,8 +43,8 @@ declare namespace globalThis {
         result: string;
         position: number;
       };
-      forgivingBase64Encode(data: Uint8Array): string;
-      forgivingBase64Decode(data: string): Uint8Array;
+      forgivingBase64Encode(data: string): string;
```

Comment:
It's a bit of a shame that the strings (presumably) have to be copied. I wonder if it would be possible to have an optional `&str` serde?